### DISCUSSION TITLE
Wir müssen Reden am 05.11.2021 20:00 Uhr

### DISCUSSION THREAD

**Discussion by Gallimathias** [OctoAwesome/octoawesome#286] [Announcements]
Wie geht es mit OctoAwesome weiter.

Diskutiere mit uns und der Community gemeinsam am 05.11.2021 um 20:00 Uhr live auf Twitch & YouTube.

Ankündigungsvideo:
https://youtu.be/piHPnqZz8DI

Twitch: https://go.twitch.tv/noobdevtv
YouTube: https://www.youtube.com/c/NoobDevGed%C3%B6ns/videos

**Comment by susch19** [comment]
Finde ich gut, bin ich mit dabei
